src/solvers/refinement/string_constraint_generator_main.cpp

```diff
@@ -17,42 +17,48 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 #include <util/pointer_predicates.h>
 #include <util/ssa_expr.h>
 
+unsigned string_constraint_generatort::next_symbol_id=1;
+
```

Comment:
Personally I don't have a preference, but Peter suggested that the initialisation should be in the cpp file.